nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/pom.xml

```diff
@@ -78,6 +78,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core-http-netty</artifactId>
+            <version>1.7.0</version>
```

Comment:
Is there a reason for using this version as opposed to the more recent version `1.9.2`?

Reply:
Dependency removed, raies azure.core and cosmos versions instead